internal/cli/test.go

```diff
@@ -157,7 +165,7 @@ func testTokenCmd(cli *cli) *cobra.Command {
 		Short: "Fetch a token for the given client and API",
 		Long: `Fetch an access token for the given client.
 If --client-id is not provided, the default client "CLI Login Testing" will be used (and created if not exists).
-Additionally, you can also specify the --audience and --scope to use.`,
+Additionally, you can also specify the --audience (API Identifer) and --scope to use.`,
```

Comment:
IMO `Additionally, you can also specify` makes it sound like this isn't required.

Reply:
Good catch! Thanks